corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -233,9 +242,147 @@ hqDefine("geospatial/js/case_grouping_map",[
         }
     }
 
+    function loadCaseGroupsInExport(caseGroups) {
+        exportModelInstance.casesToExport().forEach(caseItem => {
+            const groupData = caseGroups[caseItem.caseId];
+            if (groupData !== undefined) {
+                caseItem.groupId = groupData.groupId;
+                caseItem.groupCoordinates = groupData.groupCoordinates;
+            }
+        });
+    }
+
+    function clearCaseGroupsInExport() {
```

Comment:
Similar to previous comment, could we also move this to inside the `exportModel` class itself?